docs/usage.rst

```diff
@@ -1,132 +1,4 @@
-
-Collections inside of the ``galaxy`` entry of an execution environment
-will contribute their python and system requirements to the image.
-
-Requirements from a collection can be recognized in these ways:
-
--  A file ``meta/execution-environment.yml`` references the python
-   and/or bindep requirements files
--  A file named ``requirements.txt`` is in the root level of the
-   collection
--  A file named ``bindep.txt`` is in the root level of the collection
-
-If any of these files are in the ``build_ignore`` of the collection, it
-will not work correctly.
-
-Collection maintainers can verify that ``ansible-builder`` recognizes
-the requirements they expect by using the introspect command. Example:
-
-::
-
-    ansible-builder introspect --sanitize ~/.ansible/collections/
-
-Collection Python Rules
-^^^^^^^^^^^^^^^^^^^^^^^
-
-Python requirements files are combined into a single file using the
-``requirements-parser`` library in order to support complex syntax like
-references to other files.
-
-Entries from separate collections that give the same *package name* will
-be combined into the same entry, with the constraints combined.
-
-There are several package names which are specifically *ignored* by
-``ansible-builder``, meaning that if a collection lists these, they will
-not be included in the combined file. These include test packages and
-packages that provide Ansible itself. The full list can be found in
-``EXCLUDE_REQUIREMENTS`` in the ``ansible_builder.requirements`` module.
-
-Collection System Requirement (bindep) Rules
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The ``bindep`` format provides a way of specifying cross-platform
-requirements. A minimum expectation is that collections specify
-necessary requirements for ``[platform:rpm]``.
-
-Entries from multiple collections will be combined into a single file.
-Only requirements with *no* profiles (runtime requirements) will be
-installed to the image. Entries from multiple collections which are
-outright duplicates of each other may be consolidated in the combined
-file.
-
-Example
-~~~~~~~
-
-The example in ``test/data/pytz`` requires the ``awx.awx`` collection in
-the execution environment definition. The lookup plugin
-``awx.awx.tower_schedule_rrule`` requires the PyPI ``pytz`` and another
-library to work. If ``test/data/pytz/execution-environment.yml`` file is
-given to the ``ansible-builder build`` command, then it will install the
-collection inside the image, read ``requirements.txt`` inside of the
-collection, and then install ``pytz`` into the image.
-
-The image produced can be used inside of an ``ansible-runner`` project
-by placing these variables inside the ``env/settings`` file, inside of
-the private data directory.
-
-.. code:: yaml
-
-    ---
-    container_image: image-name
-    process_isolation_executable: podman
-    process_isolation: true
+Once you have created a definition, it's time to build your Execution Environment.
```

Comment:
Are you intending to add more to this?

Is this a thing where you're trying to split instructions for writing the spec from the instructions for using the spec?

If so, I feel like it would be good to have an abbreviated start-to-finish section for "I know what collections and containers are, how do I run my playbook?".

Reply:
Yes, that's one of the reasons this is still a draft PR. I posted this to give people an idea of what's coming. Will continue working on it today.